core/modules/startup/rootwidget.js

```diff
@@ -25,6 +25,9 @@ exports.startup = function() {
 	$tw.rootWidget.addEventListener("tm-modal",function(event) {
 		$tw.modal.display(event.param,{variables: event.paramObject, event: event});
 	});
+	$tw.rootWidget.addEventListener("tm-show-switcher",function(event) {
```

Comment:
There is already the "tm-modal" event, that can be used here. I don't think there should be a new event only for this 1 config element, which I personally think will be only used once and then never again.

Reply:
I was thinking the same, but maybe I didn't understand how @Jermolene wanted to implement this message-handler